iroh/src/endpoint.rs

```diff
@@ -2471,4 +2480,30 @@ mod tests {
         assert!(dt0 / dt1 < 20.0, "First round: {dt0}s, second round {dt1}s");
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_custom_relay() -> Result {
+        let _ep = Endpoint::empty_builder(RelayMode::custom([RelayUrl::from_str(
+            "https://use1-1.relay.n0.iroh-canary.iroh.link.",
+        )?]))
+        .bind()
+        .await?;
+
+        let _ep = Endpoint::empty_builder(RelayMode::custom([
+            "https://use1-1.relay.n0.iroh-canary.iroh.link.".parse()?,
+            "https://euw-1.relay.n0.iroh-canary.iroh.link.".parse()?,
+        ]))
+        .bind()
+        .await?;
+
+        let relays = RelayMap::try_from_iter([
+            "https://use1-1.relay.n0.iroh.iroh.link/",
+            "https://euc1-1.relay.n0.iroh.iroh.link/",
```

Comment:
We are still inconsistent with our usage of `.` and `/` for the relay urls everywhere.

We should be sticking to `.`

Reply:
Ah, wasn't super clear from that. I thought you wanted to do the same thing in 2 ways ie empty builder and pass it in from an iter.